examples/local/cluster.tmpl.yaml

```diff
@@ -68,9 +68,14 @@ spec:
       etcd: "${AWS_API_HOSTED_ZONE}"
       ingress: "${AWS_INGRESS_HOSTED_ZONE}"
     vpc:
-      cidr: "10.1.5.0/24"
-      privateSubnetCidr: "10.1.5.0/25"
-      publicSubnetCidr: "10.1.5.128/25"
+      cidr: "10.1.12.0/24"
+      privateSubnetCidr: "10.1.12.0/25"
+      publicSubnetCidr: "10.1.12.128/25"
+      privateSubnets:
+      - name: gauss-worker_0
```

Comment:
Only the `routeTableName` is needed after all.

Reply:
Nice that just the route table name is needed as it keeps the TPR simpler.